src/posh-git.psm1

```diff
@@ -17,6 +17,7 @@ if ($psv.Major -lt 3 -and !$NoVersionWarn) {
 . $PSScriptRoot\GitUtils.ps1
 . $PSScriptRoot\GitPrompt.ps1
 . $PSScriptRoot\GitTabExpansion.ps1
+. $PSScriptRoot\ParamsExpansions.ps1
```

Comment:
How about renaming this file to `GitParamTabExpansion.ps1` (no need for the `s` on Params or Expansions).  I think that would be consistent with most of the other scripts.  Thanks!